src/app/core/entity-components/entity-form/entity-form.service.ts

```diff
@@ -129,7 +130,9 @@ export class EntityFormService {
     form.markAllAsTouched();
     if (form.invalid) {
       const invalidFields = this.getInvalidFields(form, schema);
-      throw new Error($localize`Fields: "${invalidFields}" are invalid`);
+      throw new InvalidFormFieldError(
+        $localize`Fields: "${invalidFields}" are invalid`
```

Comment:
If we don't want to display the form fields, we might as well remove this code.

Reply:
true. Removing the translation and adding a more dev centered error implementation